fala/management/commands/rewrite_paths_in_static_files.py

```diff
@@ -0,0 +1,45 @@
+import glob
+import re
+
+from django.conf import settings
+from django.contrib.staticfiles.storage import staticfiles_storage
+from django.core.management.base import BaseCommand
+
+from fala.settings.base import root
+
+
+class Command(BaseCommand):
+    help = "Rewrites static file paths within our compiled css file, to be aware of our storage backend"
+    css_dir = root("assets", "css")
+    js_dir = root("assets", "scripts")
+
+    regex_patterns = {
+        "css": r"url\((?P<url>\"?(?:{static_url}|\.\.)(?P<path>.*?)\"?)\)",
+        "js": r"""url\: ?['"](?P<url>(?:{static_url})(?P<path>.*?))['"]""",
```

Comment:
_not a blocker_

🙋‍♂ It may be worth to invest in a webpack prototype spike; it has plugins which do the static rewrite in CSS and JS. This guide looks like it would be useful: https://github.com/petehunt/webpack-howto#5-stylesheets-and-images.

Reply:
Yes that's a very good idea, hadn't realised it had the image-encoding feature. We would have to make sure we could set that limit higher than anything we'd encounter, as we'd need it to always do that and never use `url()`